cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/Operand.qll

```diff
@@ -45,8 +45,10 @@ class Operand extends TStageOperand {
 
   /**
    * Gets the location of the source code for this operand.
+   * By default this is where the operand is used, but some subclasses may override this
+   * using getAnyDef() if it makes more sense.
```

Comment:
```suggestion
   * using `getAnyDef()` if it makes more sense.
```
This makes the QLDoc slightly prettier in VSCode.

Reply:
I think this change was undone when you synchronized the files :)

Reply:
I guess symbolic links are out of the question because of windows, eh? :roll_eyes: